dotCMS/src/main/java/com/dotcms/content/elasticsearch/util/MyRequestRetryHandler.java

```diff
@@ -0,0 +1,45 @@
+package com.dotcms.content.elasticsearch.util;
+
+import org.apache.http.HttpEntityEnclosingRequest;
+import org.apache.http.HttpRequest;
+import org.apache.http.client.HttpRequestRetryHandler;
+import org.apache.http.client.protocol.HttpClientContext;
+import org.apache.http.conn.ConnectTimeoutException;
+import org.apache.http.protocol.HttpContext;
+
+import javax.net.ssl.SSLException;
+import java.io.IOException;
+import java.io.InterruptedIOException;
+import java.net.UnknownHostException;
+
+public class MyRequestRetryHandler implements HttpRequestRetryHandler {
+    @Override
+    public boolean retryRequest(
+            IOException exception,
+            int executionCount,
+            HttpContext context) {
+        if (executionCount >= 3) {
+            // Do not retry if over max retry count
+            return false;
+        }
+        if (exception instanceof InterruptedIOException) {
```

Comment:
you probably can do an OR if (exception instanceof InterruptedIOException || exception instanceof UnknownHostException ...

Or even better create a method that receives all exceptions.

However I think the exception express information, probably should be wrap into a new one with a status code or so